Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -128,6 +145,16 @@ private void OnMobState(EntityUid uid, ZombieComponent component, MobStateChange
 
                 // Stop random groaning
                 _autoEmote.RemoveEmote(uid, "ZombieGroan");
+
+            }
+        }
+
+        private void OnPendingMobState(EntityUid uid, PendingZombieComponent pending, MobStateChangedEvent args)
+        {
+            if (args.NewMobState == MobState.Critical)
+            {
+                // Accelerate the process of taking damage to turn into a zombie.
+                pending.InfectedSecs = Math.Max(120, pending.InfectedSecs);
```

Comment:
This changes damages in a  implicit and non-obvious way since you wouldnt assume  InfectedSecs has something to do with damage at first, either change the damage directly here or make changes around `pain_multiple` to account for this case

Reply:
Okay, though this is a different refactor than you asked for last time, where you asked me to make this based on an event. 

It's a fair point though. Perhaps I'll set a bool "wasCrit" in the event and then use that to boost the damage in the update.